packages/toolbar/package.json

```
@@ -38,7 +38,7 @@
     "@theia/workspace": "1.56.0",
     "ajv": "^6.5.3",
     "jsonc-parser": "^2.2.0",
-    "perfect-scrollbar": "1.5.5",
+    "perfect-scrollbar": "^1.5.5",
```

```suggestion
    "perfect-scrollbar": "^1.3.0",
```

Should we maybe set this to the same version as it was before?

---

Same as above.